roles/slurm_management/templates/cron_add_missing_slurm_accounts.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Search for all existing groups inside the '/groups' folder
+# that are inside /mnt/*/ folder on the SAI machines
+for _each_pfs in /mnt/*; do
+   cd ${_each_pfs}/groups
```

Comment:
This could use an
```
if [[ -e "${_each_pfs}/groups" ]]; then
  ...
fi
```
check as not all `PFS-ses` need to have a _groups_ LFS; some may only have an _apps_ or _home_ LFS.

Reply:
Will do. I thought we have it needs to be defined everywhere as I checked the vars of FD, HC and GS.